Library/Formula/graphviz.rb

```diff
@@ -43,7 +43,7 @@ class Graphviz < Formula
 
   if build.with? "bindings"
     depends_on "swig" => :build
-    depends_on :python
+    depends_on :python if MacOS.version <= :snow_leopard
```

Comment:
It's technically a sandboxing error I guess if it's attempting to write into the Python directory. It's determining the path by doing:
```python
import sys; from distutils import sysconfig; print sysconfig.get_python_lib(1,0)
```
In `config/config_python.py`, which with Homebrew's Python installed returns:
```
/usr/local/Cellar/python/2.7.11/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages
```
And without:
```
/Library/Python/2.7/site-packages
```
I'm not quite sure on the best way to fix that given it doesn't seem to offer an override. We could just patch `configure` I guess; our preferred `site-packages` should be the same whether using system or Homebrew Python I believe.

Will defer to the Python expert... @tdsmith :smiley:.